Hardware/RAM/GenericRAM.cs

```diff
@@ -1,35 +1,42 @@
-﻿/*
- 
-  This Source Code Form is subject to the terms of the Mozilla Public
-  License, v. 2.0. If a copy of the MPL was not distributed with this
-  file, You can obtain one at http://mozilla.org/MPL/2.0/.
- 
-  Copyright (C) 2012 Michael Möller <mmoeller@openhardwaremonitor.org>
-	
-*/
+﻿// This Source Code Form is subject to the terms of the Mozilla Public

```

Comment:
I've noticed quite a lot of copyright changes, I would think we should revisit this and add a single reusable copyright statement everywhere which just says copyrighted under MPL and that users should see the LICENSE file for more information (such as https://github.com/dotnet/foundation/blob/master/guidance/copyright.md). What do you think about this? I'm not sure if it's really necessary to add authors to each file as this is already covered by MPL as a Contributor (which is also visible by GitHub).

Reply:
Sounds good.
Whenever something is edited, the old copyright part can be replaced by the new one.

// Mozilla Public License 2.0
// If a copy of the MPL was not distributed with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
// Copyright (c) .NET Foundation and Contributors
// All Rights Reserved

Reply:
I would change it to LibreHardwareMonitor instead of .NET Foundation but that would be OK then 👍